docs/examples/go.md

```diff
@@ -44,9 +44,9 @@ To execute the build, we can run `earthly +build`:
 ~/workspace/earthly/examples/go ❯ earthly +build
 buildkitd | Found buildkit daemon as docker container (earthly-buildkitd)
 context | --> local context .
-+base | --> FROM golang:1.13-alpine3.11
-+base | resolve docker.io/library/golang:1.13-alpine3.11@sha256:7d45a6fc9cde63c3bf41651736996fe94a8347e726fe581926fd8c26e244e3b2 0%
-+base | resolve docker.io/library/golang:1.13-alpine3.11@sha256:7d45a6fc9cde63c3bf41651736996fe94a8347e726fe581926fd8c26e244e3b2 100%
++base | --> FROM golang:1.15-alpine3.13
++base | resolve docker.io/library/golang:1.15-alpine3.13@sha256:7d45a6fc9cde63c3bf41651736996fe94a8347e726fe581926fd8c26e244e3b2 0%
```

Comment:
I realize this is documentation, but it looks like the checksum should be `07ec52ea1063aa6ca02034af5805aaae77d3d4144cced4e95f09d62a6d8ddf0a`

Reply:
Thanks - yeah this should be fine. It still illustrates the point.